src/views/settings/view_outfit_editor.cpp

```diff
@@ -4,6 +4,12 @@
 
 namespace big
 {
+	static int range(int lower_bound, int upper_bound)
+	{
+		std::srand(std::chrono::system_clock::now().time_since_epoch().count());
+		return rand() % (upper_bound - lower_bound + 1) + lower_bound;
```

Comment:
You're settings the seed for the `std` version of rand and then here you're using the C version of `rand`.

Use `std::rand` instead of `::rand`.

Reply:
you're right, changed